server/flights.go

```diff
@@ -56,3 +57,39 @@ func (a *App) FlightHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 }
+
+// parameters: Flight numbers for 2 different flights
+func (a *App) TimeDiff(firstFlightNum, secondFlightNum string) (time.Duration, error) {
+
+	firstFlight, err := a.Queries.GetFlight(context.Background(), firstFlightNum)
```

Comment:
try to avoid doing this in two separate queries. You can use sqlc to generate a query that returns two flights based off of 2 flight numbers